### PR TITLE
Conflicts and Infractions

### PR DESCRIPTION
**Pull request by lukel43**
## Description

This is our current version of the conflicts and infractions sections of the code of conduct. Let us know what you think!

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @lukel43 
- @Yanqiao4396 
- @gvanzin-allegheny 
- @watson02 

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by pkelly55 — DISMISSED** [review]
Looks good

File: conduct.md [hunk 1]
```diff
@@ -116,32 +116,42 @@ TODO: Write an introductory paragraph.
...
+- Students should talk out the conflict amongst their teammates.
+- Students should address the conflict as soon as it arises.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not needlessly escalate the conflict.
+- Students should not refuse to resolve the conflict
```
Comment (PaigeCD) [inline conduct.md]: This line is missing its period.

File: conduct.md [hunk 2]
```diff
@@ -116,32 +116,42 @@ TODO: Write an introductory paragraph.
...
-- TODO
-- TODO
-- TODO
+#### Level 1 Infraction
+
+Students who commit one violation will be recorded by the professor and warned orally.
+
+#### Level 2 Infraction
+
+Students who commit violations 3 times or more in total will be required to have a meeting with the professor to talk about their current infractions and develop a plan to avoid future ones.
```
Comment (PaigeCD) [inline conduct.md]: I think that in "3 times or more in total", the "in total" is a bit redundant and unnecessary.

**Review by PaigeCD — CHANGES_REQUESTED** [review]
Just two small comments from me! Other than that, I think it looks good.

**Review by anthonygc — DISMISSED** [review]
I agree with Page. Everything else looks pretty good.

**Review by laurennevill — DISMISSED** [review]
After Paige's changes are addressed, this looks great! Good work guys

**Review by laurennevill — APPROVED** [review]
looks good

**Review by PaigeCD — APPROVED** [review]
Looks good!

**Review by mariakimheinert — APPROVED** [review]
Team 1, these changes look great! Thanks for all your hard work ✨ :shipit:

**Pull request merged**